webui/src/lib/components/controls.jsx

```diff
@@ -25,18 +25,18 @@ import {
 
 const defaultDebounceMs = 300;
 
-export function debounce(func, wait, immediate) {
+export const debounce = (func, wait, immediate) => {
```

Comment:
Also, I saw you added lodash, which also include an implementation for debounce.

Reply:
I agree. I just wrote that two comments ago :)